website/docs/cloud-docs/users-teams-organizations/single-sign-on/azure-ad.mdx

```diff
@@ -54,6 +54,7 @@ For more information on the listed features, visit the [Microsoft Azure AD SAML
    1. In the **Identifier** text box, paste the **Entity ID**.
    1. In the **Reply URL** text box, paste the **Reply URL**.
    1. In the **Sign-on URL** text box, type the URL: `https://app.terraform.io/session`
```

Comment:
My only question here is --> If the configuration fails, will users know it at this point? I'm wondering if this is something we should add as a note at the end of the process, for folks who are like "help! it didn't work!". 

Actually, looking at the support article, it appears as if you need to leave the **Sign-on URL** field blank in all cases. I'm not sure if it's a fallback or if it's actually just the correct way that users should do things all the time. 
Hmm, let's get someone on engineering/support who has seen this issue to comment and give some insight! I can facilitate this on slack :)